canonical-web-and-design/ubuntu.com
# Update /openstack page as per copydoc

## Done
**Waiting on logos**

- Update /openstack page as per [copydoc](https://docs.google.com/document/d/1eqHK0hc9v7Mn-XV36QRyNkJenTXgBQR_-RULKm5rZ5o/edit#)

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: https://ubuntu-com-11351.demos.haus/openstack
    - Be sure to test on mobile, tablet and desktop screen sizes
- Compare against [copydoc](https://docs.google.com/document/d/1eqHK0hc9v7Mn-XV36QRyNkJenTXgBQR_-RULKm5rZ5o/edit#)

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/4902
